src/utils/utils.js

```diff
@@ -810,7 +810,11 @@ export function isBlock(node) {
     }
     const tagName = node.nodeName.toUpperCase();
     // Every custom jw-* node will be considered as blocks.
-    if (tagName.startsWith('JW-') || tagName === 'T') {
+    if (
+        tagName.startsWith('JW-') ||
+        (node.getAttribute && node.getAttribute('contenteditable') === 'false') ||
```

Comment:
That's not technically true, you can perfectly have a non-editable node that is inline.

Reply:
Yes but when the inline node is in the editable being `contenteditable=false`, it is like a black box. Like a "void" element.
This is the reason it should be considered as a block.

Reply:
That doesn't make it a block. A block is an element that takes up a line.
For a more precise definition, see https://www.w3.org/TR/CSS2/visuren.html#block-boxes.
What you're describing doesn't fit that definition. If rather fits that of an unbreakable. "Unbreakable" is the concept we invented to define elements that act as black boxes.
I think you fix belongs elsewhere ;-)

Reply:
I agree, new pr here: https://github.com/odoo-dev/odoo-editor/pull/251